### ISSUE TITLE
Error 400 when Around Precision property is assigned

### ISSUE THREAD

**Issue by VladislavFitz** [algolia/algoliasearch-client-swift#763]
Reported by @vishalgspaceo [here](https://github.com/algolia/instantsearch-ios-examples/issues/28).

**Describe the bug 🐛**

When I try to assign a property called aroundPrecision , Query Response throws an error with code 400

**To Reproduce 🔍**

I checked by assigning as below
```swift
searcher.request.query.aroundPrecision = [.init(integerLiteral: 1000)]
```

Request of Query Would be
```
&aroundLatLng=23.0321,72.5252&aroundRadius=50000&aroundPrecision=%5B%7B%22from%22:0.0,%22value%22:1000.0%7D%5D
```

Response of Query Would be

```json
{
  "message" : "Invalid value for \"aroundPrecision\" parameter, expected valid JSON (Expecting an integer (near 1:13))",
  "status" : 400
}
```

**Comment by vishalgspaceo** [comment]
@VladislavFitz  do we have any update on it ? We have kind of urgent release . It works well with older version.

**Comment by VladislavFitz** [comment]
Hi @vishalgspaceo.
The fix is available in [this PR](https://github.com/algolia/algoliasearch-client-swift/pull/762).
Could you test your project with the `refactor/paths` branch of the API client?

**Comment by VladislavFitz** [comment]
Hi @vishalgspaceo,
Sorry, I made a mistake in my previous message, the right PR is [here](https://github.com/algolia/algoliasearch-client-swift/pull/765). 
It's already released with [version 8.13](https://github.com/algolia/algoliasearch-client-swift/releases/tag/8.13.0). 
Let me know if you face any other issue.

**Comment by vishalgspaceo** [comment]
@VladislavFitz  Thanks for the support
